examples/angular/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "angular",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "ng": "ng",
+    "nx": "nx",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "nx workspace-lint && ng lint",
+    "e2e": "ng e2e",
+    "affected:apps": "nx affected:apps",
+    "affected:libs": "nx affected:libs",
+    "affected:build": "nx affected:build",
+    "affected:e2e": "nx affected:e2e",
+    "affected:test": "nx affected:test",
+    "affected:lint": "nx affected:lint",
+    "affected:dep-graph": "nx affected:dep-graph",
+    "affected": "nx affected",
+    "format": "nx format:write",
+    "format:write": "nx format:write",
+    "format:check": "nx format:check",
+    "update": "ng update @nrwl/workspace",
+    "update:check": "ng update",
+    "workspace-schematic": "nx workspace-schematic",
+    "dep-graph": "nx dep-graph",
+    "help": "nx help"
+  },
+  "private": true,
+  "dependencies": {
+    "@nrwl/angular": "8.4.13",
+    "@angular/animations": "^8.0.0",
+    "@angular/common": "^8.0.0",
+    "@angular/compiler": "^8.0.0",
+    "@angular/core": "^8.0.0",
+    "@angular/forms": "^8.0.0",
+    "@angular/platform-browser": "^8.0.0",
+    "@angular/platform-browser-dynamic": "^8.0.0",
+    "@angular/router": "^8.0.0",
+    "core-js": "^2.5.4",
+    "rxjs": "~6.4.0",
+    "zone.js": "^0.9.1"
+  },
+  "devDependencies": {
+    "@angular/cli": "8.1.1",
+    "@nrwl/workspace": "8.4.13",
+    "@types/node": "~8.9.4",
+    "dotenv": "6.2.0",
+    "ts-node": "~7.0.0",
+    "tslint": "~5.11.0",
+    "eslint": "6.1.0",
+    "typescript": "~3.4.5",
+    "prettier": "1.16.4",
+    "@angular/compiler-cli": "^8.0.0",
+    "@angular/language-service": "^8.0.0",
+    "@angular-devkit/build-angular": "^0.800.1",
+    "codelyzer": "~5.0.1",
+    "jest-preset-angular": "7.0.0",
+    "@nrwl/jest": "8.4.13",
+    "jest": "24.1.0",
```

Comment:
remove this, and probably a whole bunch more deps 🙂

Reply:
done